.gitignore

```diff
@@ -192,3 +192,4 @@ wwwroot
 !src/Templates/**/content/**
 .template.config/
 /OrchardCore.sln.GhostDoc.xml
+/src/Templates/OrchardCore.Cms.Templates/content
```

Comment:
What is that ?

Reply:
It added some files under that folder, I guess not needed, so I ignored the files and gitignore was changed.

Reply:
What is "it" ?
```suggestion
```

Reply:
Should I remove the line?

Reply:
Apparently VS generated bin and obj files in that location